.chloggen/deprecate-apm-stats-consumer.yaml

```diff
@@ -0,0 +1,16 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: deprecation
```

Comment:
Technically a breaking change? This changelog is mostly for us tho, so I am fine with either

Reply:
That makes sense, I updated the changelog